bootstrap/bootstrap

```diff
@@ -794,7 +794,7 @@ SunOS)
 	else
 		need_sed=yes
 	fi
-	if [ "`uname -r`" = "5.11" -a -x "/usr/bin/bash" ]; then
+	if [ "`uname -r`" = "5.12" -a -x "/usr/bin/bash" ]; then
```

Comment:
This is a throwaway test branch that is simply used for functionality testing in a bulk build. It is not for public consumption or discussion. As and when this work may be proposed for integrating into pkgsrc it will be done via the usual channel, i.e. tech-pkg@pkgsrc.org